### PR TITLE
CIV-11768 Work Allocation for Case Flags

### PR DESCRIPTION
**Pull request by jeswanth-hmcts**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/CIV-11768

### Change description ###
Work Allocation for Case Flags

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/main/resources/wa-task-permissions-civil-civil.dmn [hunk 1]
```diff
@@ -341,7 +341,7 @@ else
           <text></text>
         </inputEntry>
         <inputEntry id="UnaryTests_0m1d0fv">
-          <text>"Prod"</text>
```
Comment (jeswanth-hmcts) [inline src/main/resources/wa-task-permissions-civil-civil.dmn:344]: this change will be added once testing is done  in preview

**Review by manish14836 — CHANGES_REQUESTED** [review]
Add more specific tests

**Comment by portizdev** [comment]
Resolve the conflicts and pls add more tests pls. @neeta-hmcts found a way to add specific tests.

**Comment by jeswanth-hmcts** [comment]
> Resolve the conflicts and pls add more tests pls. @neeta-hmcts found a way to add specific tests.

**Comment by jeswanth-hmcts** [comment]
added specific  test cases and resolved comments

**Review by neeta-hmcts — APPROVED** [review]

**Pull request merged**